Cargo.toml

```diff
@@ -19,14 +19,14 @@ exclude       = [
 
 [dependencies]
 log    = "0.3.1"
-nix    = { git = "https://github.com/carllerche/nix-rust", rev = "c4257f8a76" }
+nix    = "0.6.0"
 libc   = "0.2.4"
 slab   = "0.1.0"
 time   = "0.1.33"
 bytes  = "0.3.0"
 winapi = "0.2.1"
 miow   = "0.1.1"
-net2   = { version = "0.2.19", default-features = false }
+net2   = { version = ">=0.2.19,<=0.2.23", default-features = false }
```

Comment:
> Is that just the " vs the ' in Cargo.toml? Older Cargos should just ignore the cfg business and there should be sections for compatibility there otherwise.

Here's the failure without limiting to preventing going to .24: https://travis-ci.org/carllerche/mio/builds/140069153.  Are you saying that 1.6.0 should be able to parse the `cfg` if the syntax in net2 was different (using double quotes)?  If those blocks were ignored, I do not see how net2 would have all required dependencies: https://github.com/rust-lang-nursery/net2-rs/blob/master/Cargo.toml